api/applications/models/application_registration.go

```diff
@@ -61,4 +61,9 @@ type ApplicationRegistration struct {
 	//
 	// required: true
 	ConfigBranch string `json:"configBranch"`
+
+	// AcknowledgeWarnings acknowledge all warnings
+	//
+	// required: false
+	AcknowledgeWarnings bool `json:"acknowledgeWarnings,omitempty"`
```

Comment:
Add ApplicationRegistrationResponse struct and move AcknowledgeWarnings as field to this struct, together with ApplicationRegistration.

Reply:
I guess it expected to be `ApplicationRegistrationRequest`

Reply:
Fixed